db/migrate/20250109165511_remove_taxonomy_and_override_from_filter.rb

```diff
@@ -0,0 +1,22 @@
+class RemoveTaxonomyAndOverrideFromFilter < ActiveRecord::Migration[7.0]
+  def up
+    # remove_column :filters, :override
+    # filters = Filter.where(role_id: Role.where(origin: nil).or(Role.where(builtin: 2)))
+
+    # filters.each do |filter|
+    #   filter.enforce_inherited_taxonomies
+    #   filter.update_column(:taxonomy_search, filter.taxonomy_search)
```

Comment:
After `filter.enforce_inherited_taxonomies`, the `taxonomy_search` field already contains the right value. Why go with `update_column` instead of just saving the whole record?

Reply:
changed to `filter.save`

Reply:
@adamruzicka, wouldn't `save` cause all the validations and callbacks to be run, thus making `enforce_inherited_taxonomies` run twice whilst `update_column` will just put a new value into DB?

Reply:
Oh, yes, nevermind